doc/source/conf.py

```diff
@@ -26,9 +26,16 @@
 html_logo = ansys_logo_black
 html_theme = "ansys_sphinx_theme"
 
+html_context = {
+    "github_user": "ansys",
+    "github_repo": "ansys-sphinx-theme",
+    "github_version": "main",
+}
```

Comment:
You are missing here the docs location. See PyGeometry... If I'm not mistaken it should be:
```suggestion
html_context = {
    "github_user": "ansys",
    "github_repo": "ansys-sphinx-theme",
    "github_version": "main",
    "doc_path": "doc/source",
}
```